app/pods/components/agenda/agenda-detail/sidebar-item/component.js

```diff
@@ -47,18 +45,6 @@ export default class SidebarItem extends Component {
     return null;
   }
 
-  @action
-  onEnter() {
-    setTimeout(() => {
-      this.renderDetails = true;
-    }, 100);
-  }
-
-  @action
-  onExit() {
-    this.renderDetails = false;
-  }
-
```

Comment:
Waarom is deze eruit ? dit was een fix dat de detailview ineens al deze data gaat ophalen voor alle agendapunten, ook al zijn die niet in view